ci/test_wheel_cugraph-dgl.sh

```diff
@@ -11,6 +11,11 @@ python_package_name=$(echo ${package_name}|sed 's/-/_/g')
 mkdir -p ./dist
 RAPIDS_PY_CUDA_SUFFIX="$(rapids-wheel-ctk-name-gen ${RAPIDS_CUDA_VERSION})"
 
+# Download wheels built during this job.
```

Comment:
Thanks for fixing this.  It's unfortunate that we have to do it this way vs. something closer to what we do for conda testing (add a channel pointing to the local builds).  I haven't verified one way or the other, but I'm curious if we have similar problems in our other wheel test environments, seems like we would.

Reply:
When I set up wheels originally I did this for every package that needs it. I know cugraph added a bunch of wheels afterwards, so it's probably worth an audit.

I don't think this is all that different from the conda case. The main difference is that there are multiple install lines. That was initially there to avoid needing to use `--pre` to force installation of pre-releases, but actually with the newer versioning strategies that I put in place in wheels that probably won't be an issue any more, i.e. it's untested but we ought to be able to use a single install command here now.